pkg/cluster/k8sres_test.go

```diff
@@ -864,6 +864,31 @@ func testEnvs(cluster *Cluster, podSpec *v1.PodTemplateSpec, role PostgresRole)
 	return nil
 }
 
+func testNodeAffinity(cluster *Cluster, podSpec *v1.PodTemplateSpec) error {
```

Comment:
can you add a unit test where this function is actually used :smiley:

Reply:
Facepalm. :rofl: 
Fixed. 
btw, I found there are a bunch of functions like that in that file that are not used in actual tests.

Reply:
Thanks. Yes, we found that, too :smiley: